components/AddEventForm/index.js

```diff
@@ -101,13 +102,13 @@ export default function AddEventForm({ onSubmit, appointment }) {
           type="date"
           name="date"
           defaultValue={appointment ? appointment.date : null}
-          required
+          required={selectedType === "wishlist" ? false : true}
           min="2020-01-01"
           max="2030-12-31"
         ></input>
```

Comment:
input is self-closing:

```suggestion
        />
```

Reply:
Done